app/labor/rate_limit_checker.rb

```diff
@@ -1,11 +1,16 @@
 class RateLimitChecker
   attr_reader :user, :action
 
+  def self.daily_account_follow_limit
+    ENV["RATE_LIMIT_FOLLOW_COUNT_DAILY"]&.to_i || 500
```

Comment:
this should use `ApplicationConfig["RATE_LIMIT_FOLLOW_COUNT_DAILY"]` and not `ENV` :)